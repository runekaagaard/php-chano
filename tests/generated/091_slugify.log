
---- EXPECTED OUTPUT
123
---- ACTUAL OUTPUT
Fatal error: Call to undefined method DtlIter::slugify() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/django-generated/091_slugify.php on line 4
---- FAILED
