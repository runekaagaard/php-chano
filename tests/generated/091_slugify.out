Fatal error: Call to undefined method DtlIter::slugify() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/generated/091_slugify.php on line 4